### DISCUSSION TITLE
Landscape setting default back after exiting program

### DISCUSSION THREAD

**Discussion by OrionSL78** [Stellarium/stellarium#3412] [Q&A]
So Ive installed the latest version 23.2  and everytime I exit the app and open it again my previous landscape settings default back to the original.   Everything defaults back to "Zero Horizon" and minimal brightness goes back to default.

Im running Win 11 64 bit..  Ive unistalled and reinstalled also went to 23.1 version and the landscape settings still default back.  Am I missing something?

**Comment by alex-w** [comment]
Did you saving settings after changes as default settings?

**Reply by OrionSL78** [reply]
Thanks, I wasnt aware the Save settings was in a different section

**Reply by gzotti** [reply]
The landscape section has its own "Use as default" checkbox openly visible. Also the location panel has its "Use as default" checkbox. And make sure you have disabled "Get location from network".
